app/config/question-bank.js

```diff
@@ -616,17 +614,54 @@ const questionBank = {
           answers: [
             {
               key: 'smaller-abattoir-A1',
-              value: 'Yes'
+              value: 'Yes',
+              yarKeysReset: ['fruitStorage']
             },
             {
               key: 'smaller-abattoir-A2',
               value: 'No',
-              redirectUrl: 'project-items',
+              redirectUrl: 'fruit-storage',
               yarKeysReset: ['otherFarmers']
             }
           ],
           yarKey: 'smallerAbattoir'
         },
+        {
+          key: 'fruit-storage',
+          order: 70,
+          title: 'Do you want to build new controlled atmosphere storage for top fruit?',
+          hint: {
+            text: 'Fruit that grows on trees, for example apples, pears, quinces, medlars, plums, peaches, apricots and cherries'
+          },
+          url: 'fruit-storage',
+          baseUrl: 'fruit-storage',
+          backUrl: 'smaller-abattoir',
+          nextUrl: 'solar-PV-system',
```

Comment:
is this supposed to be project-items page? if not and should be solar-PV-system, does that mean the smaller-abattoir next url needs to change?